osbs/api.py

```diff
@@ -311,6 +311,9 @@ def _create_build_directly(self, build_request, unique=None):
         build_json['spec']['serviceAccount'] = 'builder'
 
         builder_img = build_json['spec']['strategy']['customStrategy']['from']
+        build_name = builder_img['name']
+        build_kind = builder_img['kind']
```

Comment:
Can't we use the existing kind assignment here instead?

Reply:
if you mean what was there before, no because if there was imagestreamtag it will get changed int dockerimage, and I need the original

Reply:
Right. Ignore this one, then.